docs/MigrationGuide.md

```diff
@@ -16,8 +16,8 @@ To see biggest differences please consult the [changelog](Changelog.md).
   and also requires restarting TaskManager when changing the value.
 * [#1361](https://github.com/TouK/nussknacker/pull/1361) Lazy variables are removed, you should use standard enrichers for those cases.
   Their handling has been source of many problems and they made it harder to reason about the exeuction of process.   
-* [#1373](https://github.com/TouK/nussknacker/pull/1373) Use plain config to create `ClassLoaderModelData`,
-  wrapping with ModelConfigToLoad by yourself is not needed.
+* [#1373](https://github.com/TouK/nussknacker/pull/1373) Creating `ClassLoaderModelData` directly is not allowed, use
+  `ModelData.apply` with plain config, wrapping with ModelConfigToLoad by yourself is not needed.
```

Comment:
This comment is outdated after removal of `ModelConfigToLoad`

Reply:
I meant that after updating to newer NK versions, wrapping config with `ModelConfigToLoad` is not needed anymore :)

Reply:
ahh indeed. my misunderstanding